### ISSUE TITLE
[Bug]: htmlAttribute support vue,jsx,html attr

### ISSUE THREAD

**Issue by kola-web** [chrisgrieser/nvim-various-textobjs#35]
### Bug Description

The current htmlAttribute match is not robust enough

### Relevant Screenshot

vue
Need to match these six
`vue
<div v-if="false" @click="handleClick" :name="name" v-if='false' @click='handleClick' :name='name'></div>
`

jsx 
Need to match these one
`jsx
<div name={name}></div>
`

### Reproduction & Sample Text

before: <div v-if="false"></div>
after: <div v-></div>
expected: <div ></div>

### neovim version

```Text
NVIM v0.9.0
Build type: Release
LuaJIT 2.1.0-beta3

     系统 vimrc 文件: "$VIM/sysinit.vim"
         $VIM 预设值: "/opt/homebrew/Cellar/neovim/0.9.0/share/nvim"

Run :checkhealth for more info
```

### Make sure you have done the following

- [X] I have updated to the latest version of the plugin.

**Comment by kola-web** [comment]
@chrisgrieser I actually wrote these three regular expressions to cover the situation I described above, but I'm not sure how to add him to the plugin
(([@:%-_%w]+=["]).-(["])) 
(([@:%-_%w]+=[{]).-([}])) 
(([@:%-_%w]+=[']).-([']))

**Comment by chrisgrieser** [comment]
hmm, that will be a bit tricky, since [the current implementation of the textobj search algorithm](https://github.com/chrisgrieser/nvim-various-textobjs/blob/main/lua/various-textobjs/charwise-textobjs.lua#L29) only accepts one pattern. 

So I implementing multiple patterns would be a bit more work since it requires rewriting that function…

**Comment by chrisgrieser** [comment]
I have implemented searching for multiple patterns. However, since the post is lacking any examples ("needs to match these six" does not have examples following, sample text is missing, …), I cannot do anything about it, and this issue is inactionable. I am therefore closing this.

If this is still relevant, please open a new issue, without skipping all necessary steps of the issue there.
